python/tvm/relay/__init__.py

```diff
@@ -37,6 +37,7 @@
 from . import debug
 from . import param_dict
 from .backend import vm
+from .dnnl_query_layout import AutoQuery
```

Comment:
I'm not sure if they are allowed to adding this in top front. Is it possible to adding this import when BYOC=dnnl is enabled?

Reply:
move AutoQuery into relay/op/contrib/dnnl.py, and add test case in test_pass_alter_op_layout.py